test/e2e/specs/editor/various/popovers.spec.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'new editor state', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activatePlugin(
+			'gutenberg-test-plugin-post-formats-support'
+		);
+	} );
+
+	test.beforeEach( async ( { pageUtils } ) => {
+		await pageUtils.createNewPost();
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await requestUtils.deactivatePlugin(
+			'gutenberg-test-plugin-post-formats-support'
+		);
+	} );
+
+	test.describe( 'dropdown', () => {
+		test( 'toggles via click', async ( { page } ) => {
+			const isMoreMenuOpen = page.locator( 'role=menu[name="Options"i]' );
```

Comment:
Let's rename this variable to better reflect the element it queries.

```suggestion
			const moreMenu = page.locator( 'role=menu[name="Options"i]' );
```